Components/Transform/ViewOps/TestSuite/CMakeLists.txt

```diff
@@ -274,6 +310,13 @@ if(QUICC_USE_MPI)
         ViewTransposeMpiCommTests.cpp
     )
 
+    # if(TARGET QuICC::Cuda)
```

Comment:
No working or not required?

Reply:
uhmm I don't remember why is commented out, maybe it was because on my local machine the cuda aware mpi is broken, uncommenting.